docs/pages/attributes.rst

```diff
@@ -0,0 +1,215 @@
+
+
+`Scene Attributes`_
+===================
+Scene Attributes templates hold relevant information describing a scene's mesh, geometry and physical properties.  These files should be named in the format : 
+
+ 	<scenename>.scene_config.json
+	
+Scene Mesh Handles And Types
+----------------------------	
+
+Below are the handles and descriptors for various mesh assets used by a scene.
+	
+Mesh types are included to support future format flexibility. In the JSON they are limited to string keys used by :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap`, which map to values in :ref:`AssetType`.  Any mesh type specification used in JSON but not found in :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap` results in that mesh type being set to :ref:`AssetType::UNKNOWN`.  JSON-Specified mesh types will override any default settings, such as those that might be derived from file name extensions.
+
+"render mesh" 
+	- string
+	- The name of the file describing the render mesh to be used by the scene. 
+"render mesh type"
+	- string
+	- The type of the render mesh. If not specified, value will be derived based on given **"render mesh"** file extension. 
+"collision mesh"
+	- string
+	- The name of the file describing the collision mesh to be used by the scene. 
+"collision mesh type"
+	- string
+	- The type of the collision mesh. If not specified, value will be derived based on given **"collision mesh"** file extension.  Currently only "mp3d" is supported, other values default to :ref:`AssetType::UNKNOWN` 
+"semantic mesh"
+	- string
+	- The name of the file describing the scene's semantic mesh. 
+"semantic mesh type"
+	- string
+	- The type of the render mesh. If not specified, value will be derived based on given **"semantic mesh"** file extension.  Currently only "mp3d" is supported, other values default to :ref:`AssetType::INSTANCE_MESH`
+"house filename"
+	- string
+	- The name of the file containing semantic type maps and hierarchy. 
+"nav mesh"
+	- string
+	- The name of the file describing the NavMesh. 
+
+
+Scene Frame and Origin
+----------------------
+
+The tags below are used to build a coordinate frame for the scene, and will override any default values set based on render mesh file name/extension.  If either **"up"** or **"front"** are specified, both must be provided and they must be orthogonal.
+  
+"up"
+	- 3-vector
+	- Describes the **up** direction for the scene.
+"front"
+	- 3-vector
+	- Describes the **forward** direction for the scene.
+"origin"
+	- 3-vector
+	- Describes the **origin** of the scene, for alignment purposes.
+	 
+Scene Physics and Object-related Parameters
+-------------------------------------------
+
+Below are scene-specific physical and object-related quantities.  These values will override similarly-named values specified in the Physics Manager Attributes.
+
+"scale"
+	- 3-vector
+	- The default scale to be used for the scene.
```

Comment:
What does this mean? Does is the scale the background render mesh based on the value here?

Reply:
I'm curious if it will also scale all the objects you add to the scene, or just the background.

Reply:
Currently each object's scale is specified independently of the scene.  

I think it would be a reasonable addition to have a boolean "override scene scale" set at the object level (coupled with the default behavior being that every object inherits the scene's scale), or else "all objects inherit scene scale", set at the static scene level.  

Depends on the desired default/"99.4%" behavior we wish to have.